projects/kubernetes-csi/livenessprobe/docker/windows/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BUILDER_IMAGE
```

Comment:
I introduced a new dockerfile instead of reusing mainly because we had put the dockerfiles in a linux folder from day 1, not 100% sure why, but it seemed natural to introduce a windows folder.